app/components/SignupForm/SignupForm.jsx

```diff
@@ -0,0 +1,73 @@
+import { Form, useActionData, useSubmit } from "@remix-run/react";
+import { useForm } from "react-hook-form";
+import styles from "./SignupForm.css?url";
+
+export default function SignupForm() {
+  const { 
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({ mode: "onChange" });
+
+  const submit = useSubmit();
+  const actionData = useActionData();
+
+  const onSubmit = (data) => {
+    const formData = new FormData();
+    formData.append("email", data.email);
+    formData.append("password", data.password);
+    submit(formData, { method: "post" });
+  };
+
+  return (
+    <Form onSubmit={handleSubmit(onSubmit)} className="container">
+      <label htmlFor="email">Email</label>
+      <input 
+        type="email" 
```

Comment:
Try to use validation schema
https://stackoverflow.com/questions/63346815/how-can-i-use-yup-validationschema-with-react-hook-form